pkg/agent/proxy/metrics/metrics.go

```diff
@@ -0,0 +1,117 @@
+package metrics
+
+import (
+	"sync"
+
+	kmetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog"
+)
+
+var (
+	once sync.Once
+
+	subsystemProxy = "antrea_proxy"
+
+	SyncProxyLatency = kmetrics.NewHistogram(
+		&kmetrics.HistogramOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v4"},
```

Comment:
I feel like the label should be `"ip_family"` to be consistent with metric names and existing labels (`antrea_agent_ovs_flow_count` has label `"table_id"`). Unless there is a convention to use camel case for const labels?

Reply:
Thanks for mentioning it, update.